template/rwa-onboarding/Goerli-DssSpell.t.sol

```diff
@@ -20,466 +18,858 @@ pragma solidity 0.6.12;
+        assertEq(vest.res(1), 0);
+        assertEq(vest.tot(1), WAD);
+        assertEq(vest.rxd(1), 0);
 
-        uint256 castTime = block.timestamp + pause.delay();
-        hevm.warp(castTime);
+        uint256 prevBalance = dai.balanceOf(address(pauseProxy));
+        assertTrue(tryVest(address(vest), 1));
+        assertEq(dai.balanceOf(address(pauseProxy)), prevBalance + WAD);
 
-        cureSpell.cast();
+        assertEq(vest.rxd(1), WAD);
     }
 
-    function testSpellIsCast_RWA999_INTEGRATION_TELL_CULL() public {
-        if (!spell.done()) {
-            vote(address(spell));
-            scheduleWaitAndCast(address(spell));
-            assertTrue(spell.done());
-        }
-        assertTrue(oracle.good("RWA999-A"));
+    // RWA tests
+    address RWA999_A_OPERATOR                  = addr.addr("RWA999_A_OPERATOR");
+    address RWA999_A_CUSTODY                   = addr.addr("RWA999_A_CUSTODY");
+    
+    RwaLiquidationLike oracle                  = RwaLiquidationLike(addr.addr("MIP21_LIQUIDATION_ORACLE"));
 
-        tellSpell = new TellSpell();
-        vote(address(tellSpell));
+    GemAbstract          rwagem_999            = GemAbstract(addr.addr("RWA999"));
+    GemJoinAbstract      rwajoin_999           = GemJoinAbstract(addr.addr("MCD_JOIN_RWA999_A"));
+    RwaUrnLike           rwaurn_999            = RwaUrnLike(addr.addr("RWA999_A_URN"));
+    RwaOutputConduitLike rwaconduitout_999     = RwaOutputConduitLike(addr.addr("RWA999_A_OUTPUT_CONDUIT"));
+    GemAbstract          psmGem                = rwaconduitout_999.gem();
+    RwaInputConduitLike  rwaconduitinurn_999   = RwaInputConduitLike(addr.addr("RWA999_A_INPUT_CONDUIT_URN"));
+    RwaInputConduitLike  rwaconduitinjar_999   = RwaInputConduitLike(addr.addr("RWA999_A_INPUT_CONDUIT_JAR"));
+    uint256 daiPsmGemDiffDecimals              = 10**sub(dai.decimals(), psmGem.decimals());
 
-        tellSpell.schedule();
+    string OLDDOC = "";
+    string NEWDOC = "";
 
-        uint256 castTime = block.timestamp + pause.delay();
-        hevm.warp(castTime);
+    function testDocChange() private { // make public to use
+        bytes32 ilk = "RWA999-A";
 
-        tellSpell.cast();
+        (string memory docOld, address pipOld, uint48 tauOld, uint48 tocOld) =
+            oracle.ilks(ilk);
 
-        assertTrue(oracle.good("RWA999-A"));
-        hevm.warp(block.timestamp + 2 weeks);
-        assertTrue(!oracle.good("RWA999-A"));
+        assertEq(docOld, OLDDOC, "bad old document");
+
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        (string memory docNew, address pipNew, uint48 tauNew, uint48 tocNew) =
+            oracle.ilks(ilk);
+
+        assertEq(docNew, NEWDOC,     "bad new document");
+        assertEq(pipOld, pipNew,     "pip is the same");
+        assertTrue(tauOld == tauNew, "tau is the same");
+        assertTrue(tocOld == tocNew, "toc is the same");
+    }
```

Comment:
This was specific to HVB as well, should probably not be in the template.

Reply:
done